app/views/checkout/payment.scala.html

```diff
@@ -1,16 +1,21 @@
-@(form: Form[model.SubscriptionData])
+@(form: Form[model.SubscriptionData], userIsSignedIn: Boolean)(implicit request: Request[_])
```

Comment:
I would prefer to inject only the necessary data in this template, as the request has a wide scope that might make understanding the purpose of the unit harder.